typescript/sdk/src/deploy/HyperlaneAppChecker.ts

```diff
@@ -101,6 +109,8 @@ export abstract class HyperlaneAppChecker<
               name,
               expected: expectedAdmin,
               actual: actualAdmin,
+              proxyAdmin: proxyAdminContract,
```

Comment:
could we put "expected" in the `proxyAdmin` and `proxyAdminContract` names so this is a little more clear at a glance?